.gitignore

```diff
@@ -0,0 +1,6 @@
+'~'
+install
+memory_profiler
+upgrade
+TestScripts/__pycache__
+TestScripts/.pytest_cache
```

Comment:
the content of the Results folder should also be in gitignore, as they are not meant to be stored long term and/or in github.
Maybe the output folder can be set in configuration instead of writing to the repo folder?